app/src/main/java/com/x8bit/bitwarden/ui/platform/components/listitem/BitwardenListItem.kt

```diff
@@ -186,7 +186,7 @@ fun BitwardenListItem(
 data class SelectionItemData(
     val text: String,
     val onClick: () -> Unit,
-    val testTag: String? = null,
+    val testTag: String? = "AlertSelectionOption",
```

Comment:
Why can't we just apply this in the composable directly and remove this property just like you did with the `OverflowMenuItemData`